app/controller/FileList.php

```diff
@@ -18,7 +18,16 @@ public function index($request) {
         $test_directories = \app\lib\Library::retrieve('test_directories');
         $valid_dir = false;
         foreach ( $test_directories as $test_directory ) {
-            if ( strpos($dir, realpath($test_directory)) === 0 ) {
+            if(isset($test_directory['path']))
+            {
```

Comment:
Nice opinion, but I need two different behaviors for two different scenarios.

if $test_directory will be array
realpath($test_directory) fails

if $test_directory will be string
realpath($tesst_directory['path']) made notice